CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add sorting by `CREATED_AT` field. #10911 by @zedzior
   - Affected types: GiftCard, Page.
   - Deprecated `CREATION_DATE` sort field on Page type. Use `CREATED_AT` instead.
+- Add ability to sort and filter with categories.products - #10917 by @yemeksepeti-cihankarluk, @ogunheper
```

Comment:
This would have to be moved to 3.9 section, as 3.8 was already released.

Reply:
This has been moved to 3.9 section @maarcingebala